docs/openapi-ts/clients/nuxt.md

```diff
@@ -1,6 +1,6 @@
 ---
-title: Nuxt v3 Client
-description: Generate a type-safe Nuxt v3 client from OpenAPI with the Nuxt client for openapi-ts. Fully compatible with validators, transformers, and all core features.
+title: Nuxt Client
```

Comment:
Why make these changes? Once we finalize this page, we should have a v4 page too, it's ambiguous which version it's compatible with otherwise

Reply:
Nice work! I will still want to polish some of the wording in this copy, but overall I agree with the changes you've made/are making